src/cls/_ZPM/Repo/Local/Definition.cls

```diff
@@ -1,4 +1,4 @@
-Class %ZPM.PackageManager.Client.LocalServerDefinition Extends %ZPM.PackageManager.Client.ServerDefinition
```

Comment:
K: This entire class can be deleted

Reply:
@isc-kiyer  Q: This class is used as the definition of (local cache) repo, which seems to be useful if other instances want to use this namespace as a repo to find modules, using register lifecycle phase. Should we keep it?

Reply:
Done. See commit "remove local.definition and its logic"